cardano-cli/src/Cardano/CLI/Byron/Parsers.hs

```diff
@@ -636,7 +636,7 @@ parseK =
     <$> parseIntegral "k" "The security parameter of the Ouroboros protocol."
 
 parseNewDirectory :: String -> String -> Parser NewDirectory
-parseNewDirectory opt desc = NewDirectory <$> parseFilePath opt desc
```

Comment:
This used `parseFilePath` which uses `"file"` as the bash completer, which is wrong.